sdk/tests/conformance2/uniforms/incompatible-texture-type-for-sampler.html

```diff
@@ -69,69 +69,71 @@
+  t[gl.RGB10_A2]           = { textureFormat: gl.RGBA,             signed: false, int: true,  normalized: true,  depth: false, bytesPerElement: [4],        type: [gl.UNSIGNED_INT_2_10_10_10_REV], };
+  t[gl.RGBA16F]            = { textureFormat: gl.RGBA,             signed: true,  int: false, normalized: false, depth: false, bytesPerElement: [16, 8],    type: [gl.FLOAT, gl.HALF_FLOAT], };
+  t[gl.RGBA32F]            = { textureFormat: gl.RGBA,             signed: true,  int: false, normalized: false, depth: false, bytesPerElement: [16],       type: [gl.FLOAT], };
+  t[gl.RGBA8UI]            = { textureFormat: gl.RGBA_INTEGER,     signed: false, int: true,  normalized: false, depth: false, bytesPerElement: [4],        type: [gl.UNSIGNED_BYTE], };
+  t[gl.RGBA8I]             = { textureFormat: gl.RGBA_INTEGER,     signed: true,  int: true,  normalized: false, depth: false, bytesPerElement: [4],        type: [gl.BYTE], };
+  t[gl.RGB10_A2UI]         = { textureFormat: gl.RGBA_INTEGER,     signed: false, int: true,  normalized: false, depth: false, bytesPerElement: [4],        type: [gl.UNSIGNED_INT_2_10_10_10_REV], };
+  t[gl.RGBA16UI]           = { textureFormat: gl.RGBA_INTEGER,     signed: false, int: true,  normalized: false, depth: false, bytesPerElement: [8],        type: [gl.UNSIGNED_SHORT], };
+  t[gl.RGBA16I]            = { textureFormat: gl.RGBA_INTEGER,     signed: true,  int: true,  normalized: false, depth: false, bytesPerElement: [8],        type: [gl.SHORT], };
+  t[gl.RGBA32I]            = { textureFormat: gl.RGBA_INTEGER,     signed: true,  int: true,  normalized: false, depth: false, bytesPerElement: [16],       type: [gl.INT], };
+  t[gl.RGBA32UI]           = { textureFormat: gl.RGBA_INTEGER,     signed: false, int: true,  normalized: false, depth: false, bytesPerElement: [16],       type: [gl.UNSIGNED_INT], };
 
   // Sized Internal
-  t[gl.DEPTH_COMPONENT16]  = { textureFormat: gl.DEPTH_COMPONENT, colorRenderable: true,  textureFilterable: false, bytesPerElement: [2, 4],     type: [gl.UNSIGNED_SHORT, gl.UNSIGNED_INT], };
-  t[gl.DEPTH_COMPONENT24]  = { textureFormat: gl.DEPTH_COMPONENT, colorRenderable: true,  textureFilterable: false, bytesPerElement: [4],        type: [gl.UNSIGNED_INT], };
-  t[gl.DEPTH_COMPONENT32F] = { textureFormat: gl.DEPTH_COMPONENT, colorRenderable: true,  textureFilterable: false, bytesPerElement: [4],        type: [gl.FLOAT], };
-  t[gl.DEPTH24_STENCIL8]   = { textureFormat: gl.DEPTH_STENCIL,   colorRenderable: true,  textureFilterable: false, bytesPerElement: [4],        type: [gl.UNSIGNED_INT_24_8], };
-  t[gl.DEPTH32F_STENCIL8]  = { textureFormat: gl.DEPTH_STENCIL,   colorRenderable: true,  textureFilterable: false, bytesPerElement: [4],        type: [gl.FLOAT_32_UNSIGNED_INT_24_8_REV], };
+  t[gl.DEPTH_COMPONENT16]  = { textureFormat: gl.DEPTH_COMPONENT,  signed: false, int: true,  normalized: true,  depth: true, bytesPerElement: [2, 4],     type: [gl.UNSIGNED_SHORT, gl.UNSIGNED_INT], };
```

Comment:
Also consider removing `foo.depth` in favor of `IsDepth(foo.textureFormat)`.

Reply:
I think I prefer having this information inline instead of in a separate array.